src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -415,6 +412,26 @@ void cannotCreateTitleLevelRequestForUnknownInstance(String tlrFeatureStatus,
     assertThat(postResponse.getJson(), hasErrorWith(hasMessage("Instance does not exist")));
   }
 
+  @ParameterizedTest
+  @CsvSource({"Page", "Hold", "Recall"})
+  void cannotCreateTitleLevelRequestForUnknownInstance(String requestType) {
+    configurationsFixture.enableTlrFeature();
+
+    UUID patronId = usersFixture.charlotte().getId();
+    final UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+
+    Response postResponse = requestsClient.attemptCreate(new RequestBuilder()
+      .withRequestType(requestType)
+      .withRequestLevel("Title")
```

Comment:
There are methods titleRequestLevel() and itemRequestLevel() in this builder

Reply:
replaced